scripts/release/release.sh

```diff
@@ -139,90 +144,43 @@ prepare_package() {
+	mkdir $BUILD_DIR
+	cd $BUILD_DIR
 
-	# get debian version codename
+	# get version codename
 	VERSION_CODENAME=$(grep "VERSION_CODENAME=" /etc/os-release | awk -F= {' print $2'} | sed s/\"//g)
+	if [ -z ${VERSION_CODENAME} ]; then
+		OS_ID=$(grep "^ID=" /etc/os-release | awk -F= {' print $2'} | sed s/\"//g)
+		VERSION_ID=$(grep "VERSION_ID=" /etc/os-release | awk -F= {' print $2'} | sed s/\"//g)
+		VERSION_CODENAME="$OS_ID$VERSION_ID"
+	fi
 
-	# move and install
-	cd $BASE_DIR
-	mkdir -p $BASE_DIR/$VERSION/debian/$VERSION_CODENAME
-	mv elektra_$DVERSION* *$DVERSION*.deb elektra_$VERSION.orig.tar.gz $BASE_DIR/$VERSION/debian/$VERSION_CODENAME/
-	# sudo dpkg -i $BASE_DIR/$VERSION/debian/$DVERSION/*$(dpkg-architecture -qDEB_BUILD_ARCH).deb
-
-	# kdb --version | tee ~elektra/$VERSION/debian/version
-
-	# trace kdb calls
-	# log_strace "strace-debian-package"
-
-	# run and log tests
-	# run_log_tests "test-debian-package"
-
-}
-
-log_strace() {
-	CONTEXT=$1
-	mkdir $BASE_DIR/$VERSION/$CONTEXT
-
-	strace -o $BASE_DIR/$VERSION/$CONTEXT/mount.strace kdb mount file.ecf user:/release_test
-	strace -o $BASE_DIR/$VERSION/$CONTEXT/file.strace kdb file user:/release_test/b
-	strace -o $BASE_DIR/$VERSION/$CONTEXT/set.strace kdb set user:/release_test/b
-	strace -o $BASE_DIR/$VERSION/$CONTEXT/get.strace kdb get user:/release_test/b
-	strace -o $BASE_DIR/$VERSION/$CONTEXT/rm.strace kdb rm user:/release_test/b
-	strace -o $BASE_DIR/$VERSION/$CONTEXT/umount.strace kdb umount user:/release_test
-}
-
-run_log_tests() {
-	CONTEXT=$1
-	mkdir $BASE_DIR/$VERSION/$CONTEXT
-
-	KDB=kdb kdb run_all -v 2>&1 | tee $BASE_DIR/$VERSION/$CONTEXT/run_all
-	check_test_amount $BASE_DIR/$VERSION/$CONTEXT/run_all
-
-	KDB=kdb-full kdb-full run_all > $BASE_DIR/$VERSION/$CONTEXT/run_all_full 2>&1
-	check_test_amount $BASE_DIR/$VERSION/$CONTEXT/run_all_full
-
-	KDB=kdb-static kdb-static run_all > $BASE_DIR/$VERSION/$CONTEXT/run_all_static 2>&1
-	check_test_amount $BASE_DIR/$VERSION/$CONTEXT/run_all_static
-}
-
-check_test_amount() {
-	TEST_LOG_PATH=$1
+	mkdir -p $BASE_DIR/$VERSION/$VERSION_CODENAME
+	$SCRIPTS_DIR/packaging/package.sh "$PACKAGE_REVISION" 2> $BASE_DIR/$VERSION/$VERSION_CODENAME/elektra_$PVERSION.build.error > $BASE_DIR/$VERSION/$VERSION_CODENAME/elektra_$PVERSION.build
 
-	# Check if there are really >=241 or >=131 tests
-	TEST_COUNT=$(tail -n 1 $TEST_LOG_PATH | awk '/^run_all RESULTS: / { print $3 }')
-	if [ "$TEST_COUNT" -lt "131" ]; then
-		printf >&2 "Only $TEST_COUNT test(s) run"
-		exit 1
-	fi
+	mv $BUILD_DIR/package/* $BASE_DIR/$VERSION/$VERSION_CODENAME/
 }
 
 install_elektra
 run_updates
-update_debian_version_number
+git tag -f $VERSION # needed by `make source-package
+update_fedora_changelog
+update_debian_changelog
 run_checks
 prepare_package
-configure_debian_package
-tar -czvf release.tar.gz $BASE_DIR/$VERSION
+configure_package
+cd $BASE_DIR
+$SCRIPTS_DIR/release/sign-packages.sh $BASE_DIR/$VERSION/$VERSION_CODENAME
+tar -czvf release.tar.gz ./$VERSION
```

Comment:
Why do we stop using `make source-package`?

Do we still have tests with the tar ball?

Reply:
> Why do we stop using make source-package?

`make source-package` is still used in the function `prepare_package()`.
The line `tar -czvf release.tar.gz ./$VERSION` only archives all artifacts (logs, tar-ball, packages) created in the release script.

> Do we still have tests with the tar ball?

The tar ball is also tested in `prepare_package()`. 

Previously `gbp` also used the source in the tar-ball to create debian packages. Should this also be done for CPack?

Reply:
> prepare_package()

Maybe rename to make_package or create_package?

> The line tar -czvf release.tar.gz ./$VERSION only archives all artifacts (logs, tar-ball, packages) created in the release script.

Ahh, ok, this clarifies everything.

> Should this also be done for CPack?

I don't think it makes much sense. The previous way was only because gpb required the tar ball.